src/control-plane/auth/cognito-auth.ts

```diff
@@ -41,7 +26,7 @@ export interface CognitoAuthProps {
 
 export class CognitoAuth extends Construct implements IAuth {
   authorizer: IAuthorizer;
-  controlPlaneIdpDetails: string;
+  controlPlaneIdpDetails: any;
```

Comment:
I thought we didn't need `controlPlaneIdpDetails` ?

Reply:
This is still needed as different providers may need to output different details.